### PR TITLE
feat: move requests from execution payload to beacon block body

### PR DESCRIPTION
**Pull request by ensi321**
### Containers
- Remove deposit, withdrawal and consolidation requests from execution payload
- Define new container `ExecutionRequests` to encapsulate the three requests
- Add `ExecutionRequests` to beacon block body
- Update Light client containers to use Deneb execution payload
### APIs
- Update definition of `engine_newPayloadV4` and `engine_getPayloadV4` to include execution requests
- Remove  `engine_getPayloadBodiesByRangeV2` and `engine_getPayloadBodiesByHashV2`
### Light Client
- Update fork upgrade flow to discard the deposit/withdrawal/consolidation requests from payload
### Spec
- Update spec test version to v1.5.0-alpha.6
- Disable electra slashing related tests and to be enabled in #7071 

Waiting for ~~next spec test release~~ and ethereum/execution-apis#587 to be finalized

This PR does not contain update to the builder flow. Will update it in the next PR.

### CODE REVIEW

File: packages/types/src/electra/sszTypes.ts [hunk 1]
```diff
@@ -256,13 +249,13 @@ export const SignedBuilderBid = new ContainerType(
...
 );
 
-export const ExecutionPayloadAndBlobsBundle = new ContainerType(
-  {
-    executionPayload: ExecutionPayload, // Modified in ELECTRA
-    blobsBundle: denebSsz.BlobsBundle,
-  },
-  {typeName: "ExecutionPayloadAndBlobsBundle", jsonCase: "eth2"}
-);
+// export const ExecutionPayloadAndBlobsBundle = new ContainerType(
```
Comment (nflaig) [inline packages/types/src/electra/sszTypes.ts]: `execution_requests` are part of bundle now, see [here](https://github.com/ethereum/builder-specs/pull/101/files#diff-2be1000fe5e3e5a9826d0c753b4acbaa496eb07148ef9b5698b8e3bc660fef89R21)

File: packages/types/src/electra/sszTypes.ts [hunk 2]
```diff
@@ -181,9 +173,10 @@ export const BeaconBlockBody = new ContainerType(
     deposits: phase0Ssz.BeaconBlockBody.fields.deposits,
     voluntaryExits: phase0Ssz.BeaconBlockBody.fields.voluntaryExits,
     syncAggregate: altairSsz.BeaconBlockBody.fields.syncAggregate,
-    executionPayload: ExecutionPayload, // Modified in ELECTRA
+    executionPayload: ExecutionPayload,
     blsToExecutionChanges: capellaSsz.BeaconBlockBody.fields.blsToExecutionChanges,
     blobKzgCommitments: denebSsz.BeaconBlockBody.fields.blobKzgCommitments,
+    executionRequests: ExecutionRequests, // New in ELECTRA:EIP7251
```
Comment (nflaig) [inline packages/types/src/electra/sszTypes.ts:179]: `BlindedBeaconBlockBody` needs to be updated with `execution_requests_root`, see [here](https://github.com/ethereum/builder-specs/pull/101/files#diff-2be1000fe5e3e5a9826d0c753b4acbaa496eb07148ef9b5698b8e3bc660fef89R57)

will be required to pass blinded data structures test
https://github.com/ChainSafe/lodestar/blob/bb40ef7eb77f9441ec66ab8f1fdf6664240bb1a3/packages/types/test/unit/blinded.test.ts#L6

File: packages/types/src/electra/sszTypes.ts (hunk 1 above)
Comment (ensi321) [reply packages/types/src/electra/sszTypes.ts]: Thanks for the heads up. Will update the builder flow in a separate PR.

**Review by nflaig — APPROVED** [review]
LGTM

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.23.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.23.0) :tada:

**Pull request merged**